lib/blacklight_cornell/cornell_catalog.rb

```diff
@@ -161,15 +161,12 @@ def index
         params[:q].gsub!('%2F','')
         params[:q].gsub!('/','')
       end
-      if params[:search_field] == 'isbn%2Fissn' or params[:search_field] == 'isbn/issn'
```

Comment:
Did you mean to drom the isbnissn here?

Reply:
Yes, this is one of the fields that I updated in CatalogController to properly match the solr parameter, rather than having to match "isbn/isssn" to "isbnissn" throughout the code. Now that the search_field is "isbnissn", this logic is no longer needed.